utilities/report_gcm_versions.py

```diff
@@ -1,23 +1,94 @@
+"""
+This script updates the license and citation information in the following files:
 
-import json, yaml, pprint
-
-DELIVERY_MODELS = [
-    'BCC-CSM2-MR',
-    'FGOALS-g3',
-    'ACCESS-ESM1-5',
-    'ACCESS-CM2',
-    'INM-CM4-8',
-    'INM-CM5-0',
-    'MIROC-ES2L',
-    'MIROC6',
-    'NorESM2-LM',
-    'NorESM2-MM',
-    'GFDL-ESM4',
-    'GFDL-CM4',
-    'NESM3',
-]
-
-DELIVERY_EXPERIMENTS = ['historical', 'ssp245', 'ssp370']
+* citations_cc0.rst
+* citations_cc_by.rst
+* citations_cc_by_sa.rst
```

Comment:
Minor suggestion from reading this "cold":
I feel like this might be a bit confusing in the future or if this is read by someone else because these files are not in the same directory as this script. Might be able to help this by putting the full path from the root of the repo, eg `data_licenses/legalcode/filename.rst`

Reply:
thanks! this actually spits out files that aren't included in the repo. I merged the RST output into the readme because the relative includes weren't working. so maybe that warrants a better "what is this file?" docstring at the top, but it's not meant to be used regularly - just very infrequent updates by project maintainers.